db/seeds.rb

```diff
@@ -354,7 +355,7 @@ def generate_objs(method, obj, n)
     prompt_field(u, :email)
     prompt_field(u, :affiliation)
     if ENV['CAS_AUTH']
-      printf '(NetID)'
+      printf 'CAS '
```

Comment:
The prompt_field method prints username for us aleady so printf'ing "CAS " ends up prompting "CAS Username" :)